plugins/status/metrics.go

```diff
@@ -60,6 +60,18 @@ var (
 		Help:    "Histogram for the bundle loading duration by stage.",
 		Buckets: prometheus.ExponentialBuckets(1000, 2, 20),
 	}, []string{"name", "stage"})
+
+	allCollectors = []prometheus.Collector{
```

Comment:
[suggestion]: Would a comment/note here along the lines of "if you add new metrics, make sure to add them here, or they won't survive status plugin reconfigure events" be worthwhile? I feel like there's a small tripping hazard lurking here, but it's not a blocker either way.